### DISCUSSION TITLE
Rescheduling Geography Committee Meetings - August and September

### DISCUSSION THREAD

**Discussion by genevieve-anderegg** [ArctosDB/arctos#7934] [General]
Hi everyone (@ArctosDB/geo-group @falco-rk @happiah-madson )

Looking ahead at the months of August and July, I will not be in the office on the first Monday of either of those months. I am gone from August 4-14 for a work trip at the Southwest Research Station (catching butterflies and moths!), and September 2nd is Labor Day this year. 
For August, we could reschedule the meeting for the week before (July 29-August 2). I don't have many commitments that week, so I could do anytime expect for morning on Tuesday and afternoon on Thursday. 
For September, I could reschedule for pretty much any time later that week (September 3-5). We could also move the meeting to the following Monday, September 9th.

I would hate to completely cancel either of these meetings, because we have a long way to go with waterbody. Apologies for not looking ahead and failing to bring this up at the July meeting. Please let me know what you think! 
Best,
Genna

**Comment by Jegelewicz** [comment]
Please add your availability

August meeting reschedule - https://www.when2meet.com/?25589304-boBZX

September meeting reschedule - https://www.when2meet.com/?25589312-QWZZr

**Comment by sharpphyl** [comment]
July 29th and September 9th make the most sense, but my calendar is wide open.

**Comment by Jegelewicz** [comment]
@sharpphyl please use the When2 Meets https://github.com/ArctosDB/arctos/discussions/7934#discussioncomment-10036131

**Reply by sharpphyl** [reply]
Done

**Comment by mkoo** [comment]
Done!

**Comment by happiah-madson** [comment]
Done!

**Comment by genevieve-anderegg** [comment]
Thank you Teresa!

**Comment by Jegelewicz** [comment]
August meeting is changed to August 2, 9AM PDT

**Comment by Jegelewicz** [comment]
September meeting is changed to September 3, 9AM PDT

**Comment by genevieve-anderegg** [comment]
Thank you so much Teresa!!

**Comment by happiah-madson** [comment]
Hey all, I am a little confused: are we meeting today at 9 PDT? It seems like the calendars moved things to Monday, but I thought we were meeting today? @genevieve-anderegg @sharpphyl @falco-rk @mkoo

**Reply by genevieve-anderegg** [reply]
Yes we are meeting at 9PDT today! This month and September we have slightly different meeting times. The calendar series for regularly scheduled meetings just got put on our calendars which is what that was!

**Comment by mkoo** [comment]
That was what I thought too. I can meet at 9 PDT today!

**Reply by genevieve-anderegg** [reply]
See you in a few!
